terracotta/drivers/raster_base.py

```diff
@@ -54,7 +58,7 @@ def _compress_ma(self,
                      arr: np.ma.MaskedArray) -> CompressionTuple:
         compressed_data = zlib.compress(arr.data)
```

Comment:
Wait, you are not using the compression level here. Was this on purpose?

Reply:
Good catch! I missed it, thanks